src/VisualStudio/VisualBasic/Impl/CodeModel/VisualBasicCodeModelService.vb

```diff
@@ -960,7 +963,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.CodeModel
                      SyntaxKind.DelegateSubStatement
                     Return DirectCast(node, DelegateStatementSyntax).WithIdentifier(identifier)
                 Case SyntaxKind.NamespaceStatement
-                    Return DirectCast(node, NamespaceStatementSyntax).WithName(SyntaxFactory.ParseName(name))
+                    Return DirectCast(node, NamespaceStatementSyntax).WithName(SyntaxFactory.IdentifierName(name))
```

Comment:
what happens if the name is dotted?

Reply:
Good question. We should add a test.

Reply:
Note: SyntaxFactory.IdentifierName(name) will work with a dotted name... but it's *super* hacky.  I would much prefer that we just add elastic trivia to the result of ParseName.

Reply:
We can change it to manually add the elastic marker for the namespace case (I'm trying to touch as little as possible with this change, but we could do the same for C# as well...).

Reply:
I would prefer the smallest *correct* change :).  Unless this is a quick hack for one branch, and it can be done correctly in the main branch :)

Reply:
@CyrusNajmabadi Oh, might have mis-phrased. I agree with you and want to make it manually add the elastic marker, for sure. The "touch as little as possible" was meant to refer to touching the C# side.

Reply:
Ah, i see!